components/CardsGrid/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import './style.scss';
+
+const Card = ({ children }) => <div className="Card">{children}</div>;
+
+const CardsGrid = ({ columns = 2, children }) => {
+  columns = columns >= 2 ? columns : 2;
+  return (
+    <div className="CardsGrid" style={{ gridTemplateColumns: `repeat(${columns}, 1fr)` }}>
+      {(Array.isArray(children) ? children : [children]).map(e => (
```

Comment:
```suggestion
      {React.Children(children).map(e => (
```

Reply:
Well but if it's just a single item in `children` it actually comes in as an object rather than an array and then the `map` breaks!

Reply:
Oh wow, never mind; totally misread your suggestion! Tried it out and [the syntax is a little different](https://react.dev/reference/react/Children), but totally works like you advertised:

```jsx
{React.Children.map(children, e => <Card>{e}</Card>)}
```

Very sick @kellyjosephprice! 🤘 (CC: @trishaprile—just updated to reflect, so you should pull down this change before you start working on this today!)